starters/serverless-apollo-contentful/README.md

```diff
@@ -91,16 +91,20 @@ CONTENTFUL_ENVIRONMENT=master
 
 2. When that's done, go to **Settings** -> **API keys**
    Click on **Generate personal token** to get the `CONTENTFUL_CONTENT_MANAGEMENT_API_TOKEN` variable
-   ![API tokens](./screenshots/api-tokens.png)
+
+   ![API tokens](https://github.com/thisdot/starter.dev/blob/main/starters/serverless-apollo-contentful/screenshots/api-tokens.png?raw=true)
 
 3. Go to **Settings** -> **General settings** to get the `CONTENTFUL_SPACE_ID`
-   ![Space ID](./screenshots/space-id.png)  
+
+   ![Space ID](https://github.com/thisdot/starter.dev/raw/main/starters/serverless-apollo-contentful/screenshots/space-id.png)
```

Comment:
The first image is raw, and the next two aren't. Is that going to cause issues?

Reply:
Let me make them all the same